packages/PreviewCard/PreviewCard.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import safeRest from '@tds/shared-safe-rest'
+import Box from '@tds/core-box'
+import Text from '@tds/core-text'
+import { componentWithName } from '@tds/util-prop-types'
+import { colorTelusPurple, colorWhiteLilac } from '@tds/core-colours'
+
+/**
+ * The PreviewCard component creates the appearance of a page snippet, and can be used in a list format.
+ * @version ./package.json
+ */
+const PreviewCard = ({ category, other, image, body, footer, ...rest }) => {
+  let header = category || other
+  if (category && other) {
+    header = `${category} \u00B7 ${other}`
+  }
+
+  let newBody = body
+  if (body.length > 70) {
+    newBody = `${body.substr(0, 70)}...`
+  }
+
+  const Anchor = styled.a`
```

Comment:
styled-components should be defined outside of a components render so that they are not created multiple times.